source/Libraries/GogLibrary/GogGameController.cs

```diff
@@ -55,10 +55,31 @@ public override void Dispose()
 
         public override void Install(InstallActionArgs args)
         {
+            if (HandleExtras())
+            {
+                return;
+            }
+
             InitiateInstall();
             StartInstallWatcher();
         }
 
+        private bool HandleExtras()
+        {
+            var shellLink = Game.Links.FirstOrDefault(x => x.Name == "shell:open");

```

Comment:
This is bad idea. Links can get easily nuked if user redownloads metadata. It should be stored in some separate data store (file) in extension's data folder.